dev/src/index.ts

```diff
@@ -640,6 +641,30 @@ export class Firestore {
     return new WriteBatch(this);
   }
 
+  /**
+   * Creates a [BulkWriter]{@link BulkWriter}, used for performing
+   * multiple writes in parallel.
+   *
+   * @returns {WriteBatch} A BulkWriter that operates on this Firestore
+   * client.
+   *
+   * @example
+   * let bulkWriter = firestore.bulkWriter();
+   *
+   * bulkWriter.create(firestore.doc('col/doc1'), {foo: 'bar'})
+   *   .then(res => {
+   *     console.log(`Added document at ${res.writeTime}`);
+   *   });
+   * bulkWriter.update(firestore.doc('col/doc2), {foo: 'bar'});
+   * bulkWriter.delete(firestore.doc('col/doc3'));
+   * await flush().then(() => {
```

Comment:
Nit: Attach a handler to all operations to have more consistency.

This should also call "close()".